### ISSUE TITLE
Nullish and optional chaining operators break on export

### ISSUE THREAD

**Issue by kredati** [wallabyjs/quokka#541]
### Issue description or question
Nullish and optional chaining operators work just fine when doing live evaluation, but as soon as you export them (or try to import a file that includes them), Quokka throws a SyntaxError.

The example below is only the simplest version.

For reference, I'm using nvm, and running node 14.11.0. According to MDN, nullish and optional chaining operators have been supported since node 14.

### Sample code
This works:
```javascript
let foo = null ?? 'foo'; //? foo
```
This does not work, throwing a SyntaxError:
```javascript
export let foo = null ?? 'foo';
```

### Quokka.js Console Output

```
​​​​​Quokka 'play.js' (node: v14.11.0)​​​​
 
./src/prelude/unsigned/play.js:2 
export let foo = ($_$w(1, 0, $_$c), undefined) ?? ($_$w(1, 1, $_$c), 'foo'); 
                                                ^ 
 
SyntaxError: Invalid or unexpected token 
    at new Promise (<anonymous>) 
```

### Code editor version

Visual Studio Code v1.49.1

### OS name and version
OSX 10.14.6

**Comment by ArtemGovorov** [comment]
By default Quokka is using [esm module](https://github.com/standard-things/esm) to process imports/exports and looks like the module doesn't support combo of `export` and nullish and optional chaining operators.

In order to get this working, you may use [node.js native ES modules support in Quokka](https://quokkajs.com/docs/configuration.html#es-modules-and-top-level-await) by adding
```
{
  "stdEsm": false,
  "nativeEsm": true
}
```
settings to your [project's Quokka config or global Quokka config](https://quokkajs.com/docs/configuration.html#quokka-file-config).

Note that native [node ES modules have some requirements](https://nodejs.org/api/esm.html#esm_enabling) (not Quokka related): so you either need `.mjs` file extension or your `package.json` file to contain a top-level field `"type"` with a value of `"module"`. You may simply run `node src/prelude/unsigned/play.js` to test if the file runs in node, and if it does, Quokka will run it too.

**Comment by kredati** [comment]
Thanks so much for the quick response. I really appreciate the help.

I did as you said, (the `.quokka` file in the project root is exactly as above), and the Quokka output on the `play.js` is now giving me a different error:
```
​​​​​Quokka 'play.js' (node: v14.11.0)​​​​
require.resolve is not a function 
```
My `package.json` includes `type: 'module'`, and `node play.js` works just fine.

**Comment by ArtemGovorov** [comment]
We can't reproduce the issue locally. 

I have created this sample repo https://github.com/ArtemGovorov/quokka-541, with the `src/prelude/unsigned/play.js` file:

```js
export let foo = null ?? 'foo';
```

and starting Quokka on it works for me:

<img width="780" alt="Screen Shot 2020-09-19 at 9 09 01 am" src="https://user-images.githubusercontent.com/979966/93652273-cad0c600-fa57-11ea-8973-8f4af6d8cc25.png">

Can you please check if the sample project is working for you, and if so, check what's different in your project that causes the issue?

**Comment by kredati** [comment]
I cloned the repo, and your sample project is not working for me. I also created a bare bones project which also did not work, with the same `require.resolve is not a function` error.

Ideas for next steps? I can certainly rewrite my code not to use the optional chaining and nullish coalescing operators. But I figure this may also be something you all might want to troubleshoot; I'm happy to work with you if you do.
